lib/assets/javascripts/cartodb/table/menu_modules/filters/histogram.js

```diff
@@ -740,17 +726,17 @@
           .append("g")
           .attr("transform", "translate(" + margin.left + "," + margin.top + ")");
 
-        g.append("clipPath")
-          .attr("id", "clip-" + id)
-          .append("rect")
-          .attr("width", width)
-          .attr("height", height);
-
-        g.selectAll(".bar")
-          .data(['background', 'foreground'])
-          .enter().append("path")
-          .attr("class", function(d) { return d + " bar"; })
-          .data([group, group])
+          g.append("clipPath")
+            .attr("id", "clip-" + id)
+            .append("rect")
+            .attr("width", width)
+            .attr("height", height);
+
+          g.selectAll(".bar")
+            .data(['background', 'foreground'])
+            .enter().append("path")
+            .attr("class", function(d) { return d + " bar"; })
+            .data([group, group])
```

Comment:
Have you changed the indentation on this?

Reply:
Yes. I Adjusted it to be more consistent with the rest of the document